src/wallet/rpcwallet.cpp

```diff
@@ -4329,9 +4341,5 @@ static const CRPCCommand commands[] =
     { "wallet",             "walletprocesspsbt",                &walletprocesspsbt,             {"psbt","sign","sighashtype","bip32derivs"} },
 };
 // clang-format on
-
-    for (unsigned int vcidx = 0; vcidx < ARRAYLEN(commands); vcidx++)
-        handlers.emplace_back(chain.handleRpc(commands[vcidx]));
+    return MakeSpan(commands);
```

Comment:
Would it be possible to do just this change in the first commit and all other changes in the second commit? If yes, that should simplify review. If no, I will take another look.

Reply:
Thanks for splitting up. This made the review easier for me.